src/Business Foundation/App/NoSeries/readme_refactoring.md

```diff
@@ -0,0 +1,134 @@
+# Refactoring to use the new No. Series module
+
+This document is created to give a brief explanation of how refactor your code to use the new No. Series module.
+This is done through examples that was also shown during directions.
```

Comment:
I don't think it matters when or where it was shown... there are many directions conferences...